app/v1/views/questions.py

```diff
@@ -2,48 +2,63 @@
 
 from flask import Blueprint, jsonify, request
 
+from app.utilities import check_keys, check_empty_dict
+from app.v1.models import Question
+
 questions = Blueprint("questions_blueprint", __name__,
-                      url_prefix="/stackoverflowlite/api/v1/questions")
+                      url_prefix="/stackoverflowlite/api/v1")
+
+question_instance = Question()
 
 
-@questions.route('/', methods=['POST'])
+@questions.route('/questions', methods=['POST'])
 def post_question():
     """method to post a new question"""
     question_info = request.get_json()
 
-    return jsonify({"Question": question_info})
-
-
-@questions.route('/', methods=['GET'])
+    # Check that all fields have been provided
+    if check_keys(question_info, 2):
```

Comment:
Change this implementation to be able to tell the user which field(s) has(have) not been provided.

Reply:
This has been implemented.